roadmap/implementors-guide/src/type-definitions.md

```diff
@@ -1,13 +1,5 @@
 # Data Structures and Types
```

Comment:
Given the rename in https://github.com/paritytech/polkadot/pull/1270/files#diff-285a34366a47d57b689b4b08d3e9681fR39 , this file should be renamed to `types/README.md`.

Reply:
The PR is still in-progress and I will just remove this file once I have a home for the remaining types within it.